pywbem/cim_operations.py

```diff
@@ -297,7 +303,8 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     using *operation recorders*, at the level of method calls and returns, and
     at the level of CIM-XML requests and responses.
     The :class:`~pywbem.LogOperationRecorder` class records the operations in
-    the Python logging facility.
+    the Python logging facility. This recorder is activated through the
+    `connect` parameter of :class:`~pywbem.WBEMConnection.configure_logger`.
```

Comment:
* The parameter is named "connection".
* It is not just this one method but also the other.

Reply:
Think I got them all.